restservice/src/test/java/org/itevents/service/sendmail/SendGridMailServiceTest.java

```diff
@@ -0,0 +1,41 @@
+package org.itevents.service.sendmail;
+
+import com.sendgrid.SendGrid;
+import static org.junit.Assert.*;
```

Comment:
IDEA prompts to do 2 imports instead import static org.junit.Assert.* like
import static org.junit.Assert.assertNotNull;
import static org.junit.Assert.assertTrue;
We dont import unused methods.